Update OEE to 10.17 and Shifts feature. (#1238)
SoftwareAG/c8y-docs: content/oee/oee-dashboards-bundle/machine-dashboard.md

#### Time period and resolution

> In the **Time range** field you can select a time range for which the OEE values will be displayed in the [**OEE status**](#oee-status) and [**OEE data**](#data) sections. By default, this will select a 24-hour period based on the current date. The back and forward arrows will move the time range based on the difference betwen the start and end time, i.e. if the difference between the two times is 4 hours, then the arrows will use steps of 4 hours. 

Reviewer: ```suggestion
In the **Time range** field you can select a time range for which the OEE values will be displayed in the [**OEE status**](#oee-status) and [**OEE data**](#data) sections. By default, this will select a 24-hour period based on the current date. The back and forward arrows will move the time range based on the difference betwen the start and end time, that is, if the difference between the two times is 4 hours, then the arrows will use steps of 4 hours. 
```

Below these values the losses caused by Availability, Performance and Quality are displayed in terms of time and amount.

<a name="data"></a>
### Data

In the **Data** section, a chart is available to display the current values of selected components over the specified time frame. Each of the specific items, such as OEE, Availability, Performance, and Quality, can be selected at the right by activating/deactivating the respective toggles. A toggle also exists to overlay alarms on the graph. In this way, it is possible to correlate the alarm event with other measurements.

![Chart view](/images/oee/dashboards/dashboard-single-view.png)

{{< c8y-admon-info >}}
The time frame does not influence the direct calculation of the OEE. It only defines the time period for which the OEE is displayed here.
{{< /c8y-admon-info >}}

{{< c8y-admon-info >}}

> Implausible values are taken into consideration for shift calculations. Most implausible values are caused by a too low Ideal Cycle Amount. Even if implausible conductivity values arise over short periods of time, this will pay off over a longer interval. In order to obtain meaningful values for a shift, it is therefore useful to calculate with implausible values.

Reviewer 2: conductivity -> productivity

{{< /c8y-admon-info >}}

> ### Pareto chart

Reviewer 2: Why did you remove the explanation?

Author: @haraldmeyer Removed it because the section view is also removed as a feature. If the explanation applies also to the single chart, I can bring it back.

Reviewer 3: Section view has been removed. Would agree to Marks change. @haraldmeyer Anything we need to change?

Author: Removed info box as directed by Thomas.
